File: 10 - Python/Exercícios/02 - Estrutura Condicional/Exercício 06.py
#Exercício - Glicose

glicose = float(input("Digite a medida de glicose: "))

if glicose <= 100:
    print("Classificação: Normal")
elif glicose <= 140:
    print("Classificação: Elevado")
else:
    print("Classificação: Diabetes")

File: 06 - C/Exercícios/02 - Estrutura Condicional/Exercício 06.c
//Exercício - Glicose

#include <stdio.h>

int main() {

    double glicose;

    printf("Digite a medida da glicose: ");
    scanf("%lf", &glicose);

    if (glicose <= 100){
        printf("Classificacao: Normal");
    }
    else if(glicose <= 140){
        printf("Classificacao: Elevado");
    }
    else{
        printf("Classificacao: Diabetes");
    }

    return 0;

}